6001-7000/LIT6936TarWaAmidBeta.xml

```diff
@@ -220,37 +221,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             አልቦቱ፡ ጥንት፡ ወኢተፍጻሜት፡ <gap reason="ellipsis"/>
                             ወረድኤትነሰ፡ እምኀበ፡ እግዚአብሔር።</q>
                     </div>
                     <div type="textpart" subtype="part" xml:id="TeyyaqeAzman">
                         <label>Discourse on the nature of time</label>
-                        <note>The following incipit and explicit are taken from <ref target="#J"/>.</note>
+                        <note>The following incipit and explicit are taken from <ref type="mss" corresp="FSUor134"/>.</note>
                         <q xml:lang="gez">ነገር፡ በእንተ፡ ጥያቄ፡ አዝማን፡ <gap reason="ellipsis"/>
                             ወመዓልት፡ ለ፲ወ፪ክፍል፡ እንተ፡ ይእቲ፡ ሰዓታት።</q>
                     </div>
                     <div type="textpart" subtype="part" xml:id="MatanaAzman">
                         <label>Discourse about the quantity of time and how much of it has elapsed</label>
-                        <note>The following incipit and explicit are taken from <ref target="#J"/>.</note>
+                        <note>The following incipit and explicit are taken from <ref type="mss" corresp="FSUor134"/>.</note>
                         <q xml:lang="gez">ነገር፡ በእንተ፡ መጠነ፡ አዝማን፡ <gap reason="ellipsis"/>
                             ለእመ፡ ሠምረ፡ እግዚአብሔር።</q>
                     </div>
                     <div type="textpart" subtype="part" xml:id="TentaFetrat">
                         <label>Discourse on the commencement of the Creation</label>
-                        <note>The following incipit and explicit are taken from <ref target="#J"/>.</note>
+                        <note>The following incipit and explicit are taken from <ref type="mss" corresp="FSUor134"/>.</note>
                         <q xml:lang="gez">ነገር፡ በእንተ፡ ጥንተ፡ ፍጥረት፡ <gap reason="ellipsis"/>
                             ይቤ፡ መጽሐፍ፡ ወአዕረፈ፡ እግዚአብሔር፡ እምኵሉ፡ ግብሩ፡ በዕለት፡ ሰብዕ፡ ወዐረካ፡ ወቀደሳ።</q>
                     </div>
                 </div>
                     
                 <div type="textpart" subtype="part" xml:id="preislamichistory">
                     <label>The pre-Islamic history from <persName ref="PRS1386Adam"/> to <persName ref="PRS5294Heracliu"/>.</label>
-                    <note>If not indicated otherwise, the following incipits or explicits of the textparts are taken from <ref target="#J"/>.</note>
+                    <note>If not indicated otherwise, the following incipits or explicits of the textparts are taken from <ref type="mss" corresp="FSUor134"/>.</note>
                     <div type="textpart" subtype="part" xml:id="Adam">
                         <label>1st Generation - Adam</label>
-                        <q xml:lang="gez">ወይቤ፡ ለእግዚአብሔር፡ አምላክ፡ ገነት፡ ውስተ፡ <app><lem>ኤዴን፡</lem><rdg corresp="J">ኤዶም፡</rdg></app>
+                        <q xml:lang="gez">ወይቤ፡ ለእግዚአብሔር፡ አምላክ፡ ገነት፡ ውስተ፡ <app><lem>ኤዴን፡</lem><rdg corresp="#J">ኤዶም፡</rdg></app>
```

Comment:
`rdg `do not take `corresp`

since it is you who long wanted to learn doing critical editions with XML I suggest once again that you study the materials provided to understand how they are structured and what the elements are, I truly believe that it would be good for you to understand the concept first.